src/shardus/shardus-types.ts

```diff
@@ -973,6 +973,8 @@ export interface ServerConfiguration {
     requiredVotesPercentage: number
     // /** a fix to prevent node from producing different ts for same txId */
     timestampCacheFix: boolean
+    // whether to use ajv for validation for CycleRecord sync
+    useAjvCycleRecordValidation: boolean
```

Comment:
I think this may not be ready to merge yet.   I don't see this flag initialized or checked.

Reply:
Sorry I missed the `Sync.ts` file commit